internal/exec/stages/disks/partitions.go

```diff
@@ -91,6 +91,17 @@ func partitionMatches(existing, spec types.Partition) error {
 	return nil
 }
 
+// partitionShouldBeInspected returns if the partition has zeroes that need to be resolved to sectors.
+func partitionShouldBeInspected(part types.Partition) bool {
+	if part.Number == 0 {
```

Comment:
Is the case where the partition number isn't specified or set to 0 ignored because they will be inspected anyways? I don't see anything that says you cannot specify 0'd `Start*` or `Size*` values when using a 0'd partition number.

Reply:
number 0 means "next unused partition number" which by definition can't exist and thus is un-inspectable.

Reply:
Fair enough.